### DISCUSSION TITLE
Display one specific bloc

### DISCUSSION THREAD

**Discussion by Ghozgul** [maruohon/litematica#392] [Ideas]
Hey, I was actually building a house for once, instead of complicated redstone, and I have a lot of different blocs everywhere, so I usually use the layer selection for most of my builds, but for this one I was thinking about something else.

My idea is to instead of using the Single Layer, only display a specific bloc. My last redstone build used like 2 stack of White concrete for the redstone dust, so it would be interesting to have an option to show only the concrete on all layer, but only the concrete.

Hope my point is clear enough to understand ^^

**Comment by maruohon** [comment]
A better system for this is planned, but you can also do that currently by removing all other blocks from the schematic using the Rebuild mode (see https://github.com/maruohon/litematica/wiki/schematic_editing) with the `schematicRebuildRemoveAllExcept` hotkey. Unfortunately in the 1.13+ versions there is no Reload button yet in the Loaded Schematics list, so you would have to re-log to reset/reload the schematic from file.

**Reply by Ghozgul** [reply]
That sounds promising, can't wait to see that =D
